README.rst

```diff
@@ -7,6 +7,9 @@ LabelImg
 .. image:: https://img.shields.io/travis/tzutalin/labelImg.svg
         :target: https://travis-ci.org/tzutalin/labelImg
 
+.. image:: https://mybinder.org/badge.svg 
+        :target: https://mybinder.org/v2/gh/4QuantOSS/labelImg/binder-tools
```

Comment:
Post pull request acceptance, this should be changed to https://mybinder.org/v2/gh/tzutalin/labelImg